js/main.js

```diff
@@ -25,11 +31,16 @@ Site = {
       $(this).html(string);
     });
   },
+
+  showArchiveImage: function(id) {
```

Comment:
this should be in it's own object as a child of `Site`. something like `Site.Editorial`. That way if we scale we don't need to refactor, and code is kept it its own place